.github/workflows/snapm.yml

```
@@ -120,4 +122,68 @@ jobs:
+  virt_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        firmware: [bios, uefi]
+        storage: [lvm, lvm-thin]
+        base_os: [fedora41, fedora42, centos-stream9, centos-stream10]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install virt dependencies
+        env:
+          DEBIAN_FRONTEND=noninteractive
```

actionlint (1.7.7)

expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)